src/LockToVotePlugin.sol

```diff
@@ -44,7 +44,7 @@ contract LockToVotePlugin is ILockToVote, MajorityVotingBase, LockToGovernBase {
 
     /// @notice Thrown when a proposal action is targeting address(0)
```

Comment:
@brickpop the error description should also be updated to mention that it will also revert when the `to` value is the `lockManager`